source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,117 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. 
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- Root level access to the server
+
+.. note:: 
+
+  With Mattermost Omnibus, the ``config.json`` file no longer exists. So you’ll need to use mmctl to make changes to your Mattermost server configuration using ``mmctl config edit``. If you wish to manage your server locally, you may want to enable local mode for mmctl.
```

Comment:
This is something that we can move to the "Configuring Mattermost Omnibus" section

Reply:
Done.